Project.toml

```diff
@@ -17,10 +17,11 @@ StatsBase = "0.24, 0.25, 0.26, 0.27, 0.28, 0.29, 0.30, 0.31, 0.32, 0.33"
 julia = "0.7, 1"
 
 [extras]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
need to specify Compat for DataFrames
else tests will break randomly at some point.

Reply:
Just add it in the `[compat]` as if it's normal deps would work.

Reply:
Added